.gitignore

```diff
@@ -51,3 +51,6 @@ analytics/filesystem/testFiles/
 # autogenerated mac file
 
 .DS_Store
+
+# generated file with modules
+modules/builder.go
```

Comment:
I would like for this file to be included in source control, such that you can download and compile or run without issue. The file only needs to be regenerated when a new module is added and we can make that part of the dev instructions for how to build a module.

Reply:
Got you, I'll change it.